src/interpreter.py

```diff
@@ -1,8 +1,11 @@
 from calc import *
 
 class Interpreter:
+
+	__variables = {}
```

Comment:
As we are using this for variable assignment why not store the corresponding values into the dictionary itself and then we can access each of values as keys.

Reply:
We do actually. And we can access values via keys like so:
variables = {'var1' : val1, 'var2' : val2}
print variables[var1]
output: val1